ENIGMAsystem/SHELL/Platforms/General/PFwindow.cpp

```diff
@@ -59,6 +59,17 @@ void input_key_up(int key) {
   keybdstatus[key]=0;
 }
 
+void input_mouse_down(int button) {
+  enigma_user::mouse_lastbutton = button;
+  enigma_user::mouse_button = button;
+  mousestatus[button] = 1;
+}
+
+void input_mouse_up(int button) {
+  enigma_user::mouse_button = 0;
```

Comment:
This is different from both the keyboard logic *and* how we were doing it before (which is *not* tracking this)...